CWE-94-Python_darwin/andy897221_Proof-of-Play-Flow-Demo/PoP.py

```
@@ -0,0 +1,107 @@
+    def __init__(self, nodeID, winnerFunc, ratingFunc, setupJSON=None):
+        from_path = path(sys.argv[0])
+
+        if os.path.isfile(f'{from_path}/config/{nodeID}.json'):
+            print(f"{nodeID} config file found.")
+        else:
+            print(f"{nodeID} config file not found, runnig initialization process...")
+            _init.init(nodeID=nodeID, from_path=from_path, setupJSON=setupJSON)
+
+        configFile = f'{from_path}/config/{nodeID}.json'
+        if not os.path.isfile(configFile):
+            raise FileNotFoundError('conifiguration file not found, please pass setupJSON when creating PoP.handler')
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)